### ISSUE TITLE
Only update every 5 minutes

### ISSUE THREAD

**Issue by madpilot** [madpilot/hass-amber-electric#6]
Update should occur at 45 seconds past the 5 minute boundary

**Comment by OmgImAlexis** [comment]
Going off experience with scale I'd suggest being careful with this that you don't cause a herding issue with everyone customers hitting this at the same time.

**Comment by madpilot** [comment]
Good point - I'll throw some randomisation in there. And hopefully by the time we hit a scaling issue from HA we'll have a better mechanism for this.
